mempool/mempool_test.go

```diff
@@ -450,6 +450,28 @@ func TestMempoolMaxMsgSize(t *testing.T) {
 
 }
 
+func TestMempoolSizeBytes(t *testing.T) {
+	app := kvstore.NewKVStoreApplication()
+	cc := proxy.NewLocalClientCreator(app)
+	mempool := newMempoolWithApp(cc)
+
+	assert.EqualValues(t, 0, mempool.SizeBytes())
+
+	err := mempool.CheckTx([]byte{0x01}, nil)
+	require.NoError(t, err)
+	assert.EqualValues(t, 1, mempool.SizeBytes())
+
+	mempool.Update(1, []types.Tx{[]byte{0x01}}, nil, nil)
+	assert.EqualValues(t, 0, mempool.SizeBytes())
+
+	err = mempool.CheckTx([]byte{0x02, 0x03}, nil)
+	require.NoError(t, err)
+	assert.EqualValues(t, 2, mempool.SizeBytes())
+
+	mempool.Flush()
+	assert.EqualValues(t, 0, mempool.SizeBytes())
+}
```

Comment:
Maybe it make sense adding a test for scenario where mempool is already full so trying to add tx will fail.

Reply:
Would be good to add this test, and also one where the tx gets removed during recheck if possible